src/components/SQFormDialog/SQFormDialogInner.js

```diff
@@ -19,12 +20,28 @@ const Transition = React.forwardRef((props, ref) => {
   return <Slide direction="down" ref={ref} {...props} />;
 });
 
+const stickyStyles = {
+  position: 'sticky',
+  background: ({palette}) => palette.background.paper,
+  zIndex: 1
+};
+
+const useTitleStyles = makeStyles({
+  root: {
+    ...stickyStyles,
+    top: 0,
+    borderBottom: ({palette}) => `1px solid ${palette.divider}`
+  }
+});
 const useActionsStyles = makeStyles({
   root: {
     display: 'flex',
     justifyContent: 'space-between',
     flex: '1 1 100%',
-    padding: '16px 24px'
+    padding: '16px 24px',
```

Comment:
We can pass a callback to makeStyles that gives us access to the theme where we could use the built in theme spacing. 

theme.spacing(2) would result in 16. Material Design uses multiples of 8.